sdk/eventgrid/eventgrid/CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Release History
 
-## 4.2.1 (Unreleased)
+## 4.3.0 (Unreleased)
+
+### New Features
+
+- Added new System Events: "Microsoft.AgFoodPlatform.ApplicationDataChanged", "Microsoft.AgFoodPlatform.BoundaryChanged", "Microsoft.AgFoodPlatform.CropChanged",
```

Comment:
Would a bulleted list take up too much space? These paragraph-style lists feel difficult to scan

Reply:
Can do!